{"error":{"name":"MongoNetworkError","errorLabels":["TransientTransactionError"]},"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\nMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:182:13)\n    at Connection.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:273:13)\n    at Connection.emit (events.js:182:13)\n    at Socket.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:273:13)\n    at Socket.emit (events.js:182:13)\n    at emitErrorNT (internal/streams/destroy.js:82:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)\n    at process._tickCallback (internal/process/next_tick.js:63:19)","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:182:13)\n    at Connection.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:273:13)\n    at Connection.emit (events.js:182:13)\n    at Socket.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:273:13)\n    at Socket.emit (events.js:182:13)\n    at emitErrorNT (internal/streams/destroy.js:82:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)\n    at process._tickCallback (internal/process/next_tick.js:63:19)","exception":true,"date":"Tue Dec 18 2018 11:45:50 GMT-0500 (Eastern Standard Time)","process":{"pid":12080,"uid":1000,"gid":1000,"cwd":"/home/michael/repos/vidly-server","execPath":"/home/michael/.nvm/versions/node/v10.14.2/bin/node","version":"v10.14.2","argv":["/home/michael/.nvm/versions/node/v10.14.2/bin/node","/home/michael/repos/vidly-server/index.js"],"memoryUsage":{"rss":70070272,"heapTotal":60067840,"heapUsed":21077288,"external":18282722}},"os":{"loadavg":[1.0498046875,1.66748046875,1.91748046875],"uptime":44465},"trace":[{"column":11,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js","function":null,"line":564,"method":null,"native":false},{"column":13,"file":"events.js","function":"Pool.emit","line":182,"method":"emit","native":false},{"column":12,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js","function":null,"line":317,"method":null,"native":false},{"column":13,"file":"events.js","function":"Object.onceWrapper","line":273,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Connection.emit","line":182,"method":"emit","native":false},{"column":50,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js","function":null,"line":246,"method":null,"native":false},{"column":13,"file":"events.js","function":"Object.onceWrapper","line":273,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Socket.emit","line":182,"method":"emit","native":false},{"column":8,"file":"internal/streams/destroy.js","function":"emitErrorNT","line":82,"method":null,"native":false},{"column":3,"file":"internal/streams/destroy.js","function":"emitErrorAndCloseNT","line":50,"method":null,"native":false},{"column":19,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":63,"method":"_tickCallback","native":false}]}
{"error":{"name":"MongoNetworkError","errorLabels":["TransientTransactionError"]},"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\nMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:182:13)\n    at Connection.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:273:13)\n    at Connection.emit (events.js:182:13)\n    at Socket.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:273:13)\n    at Socket.emit (events.js:182:13)\n    at emitErrorNT (internal/streams/destroy.js:82:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)\n    at process._tickCallback (internal/process/next_tick.js:63:19)","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:182:13)\n    at Connection.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:273:13)\n    at Connection.emit (events.js:182:13)\n    at Socket.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:273:13)\n    at Socket.emit (events.js:182:13)\n    at emitErrorNT (internal/streams/destroy.js:82:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)\n    at process._tickCallback (internal/process/next_tick.js:63:19)","exception":true,"date":"Wed Dec 19 2018 12:41:07 GMT-0500 (Eastern Standard Time)","process":{"pid":6893,"uid":1000,"gid":1000,"cwd":"/home/michael/repos/vidly-server","execPath":"/home/michael/.nvm/versions/node/v10.14.2/bin/node","version":"v10.14.2","argv":["/home/michael/.nvm/versions/node/v10.14.2/bin/node","/home/michael/repos/vidly-server/index.js"],"memoryUsage":{"rss":73342976,"heapTotal":43331584,"heapUsed":25277304,"external":18428263}},"os":{"loadavg":[2.73681640625,2.029296875,1.552734375],"uptime":134183},"trace":[{"column":11,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js","function":null,"line":564,"method":null,"native":false},{"column":13,"file":"events.js","function":"Pool.emit","line":182,"method":"emit","native":false},{"column":12,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js","function":null,"line":317,"method":null,"native":false},{"column":13,"file":"events.js","function":"Object.onceWrapper","line":273,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Connection.emit","line":182,"method":"emit","native":false},{"column":50,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js","function":null,"line":246,"method":null,"native":false},{"column":13,"file":"events.js","function":"Object.onceWrapper","line":273,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Socket.emit","line":182,"method":"emit","native":false},{"column":8,"file":"internal/streams/destroy.js","function":"emitErrorNT","line":82,"method":null,"native":false},{"column":3,"file":"internal/streams/destroy.js","function":"emitErrorAndCloseNT","line":50,"method":null,"native":false},{"column":19,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":63,"method":"_tickCallback","native":false}]}
{"error":{"name":"MongoNetworkError","errorLabels":["TransientTransactionError"]},"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\nMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:182:13)\n    at Connection.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:273:13)\n    at Connection.emit (events.js:182:13)\n    at Socket.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:273:13)\n    at Socket.emit (events.js:182:13)\n    at emitErrorNT (internal/streams/destroy.js:82:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)\n    at process._tickCallback (internal/process/next_tick.js:63:19)","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]\n    at Pool.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js:564:11)\n    at Pool.emit (events.js:182:13)\n    at Connection.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js:317:12)\n    at Object.onceWrapper (events.js:273:13)\n    at Connection.emit (events.js:182:13)\n    at Socket.<anonymous> (/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js:246:50)\n    at Object.onceWrapper (events.js:273:13)\n    at Socket.emit (events.js:182:13)\n    at emitErrorNT (internal/streams/destroy.js:82:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)\n    at process._tickCallback (internal/process/next_tick.js:63:19)","exception":true,"date":"Wed Dec 19 2018 12:42:49 GMT-0500 (Eastern Standard Time)","process":{"pid":7556,"uid":1000,"gid":1000,"cwd":"/home/michael/repos/vidly-server","execPath":"/home/michael/.nvm/versions/node/v10.14.2/bin/node","version":"v10.14.2","argv":["/home/michael/.nvm/versions/node/v10.14.2/bin/node","/home/michael/repos/vidly-server/index.js"],"memoryUsage":{"rss":67989504,"heapTotal":41758720,"heapUsed":25423248,"external":18456094}},"os":{"loadavg":[0.92529296875,1.6025390625,1.4462890625],"uptime":134284},"trace":[{"column":11,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/topologies/server.js","function":null,"line":564,"method":null,"native":false},{"column":13,"file":"events.js","function":"Pool.emit","line":182,"method":"emit","native":false},{"column":12,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/pool.js","function":null,"line":317,"method":null,"native":false},{"column":13,"file":"events.js","function":"Object.onceWrapper","line":273,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Connection.emit","line":182,"method":"emit","native":false},{"column":50,"file":"/home/michael/repos/vidly-server/node_modules/mongodb-core/lib/connection/connection.js","function":null,"line":246,"method":null,"native":false},{"column":13,"file":"events.js","function":"Object.onceWrapper","line":273,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Socket.emit","line":182,"method":"emit","native":false},{"column":8,"file":"internal/streams/destroy.js","function":"emitErrorNT","line":82,"method":null,"native":false},{"column":3,"file":"internal/streams/destroy.js","function":"emitErrorAndCloseNT","line":50,"method":null,"native":false},{"column":19,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":63,"method":"_tickCallback","native":false}]}
